doc/whatsnew/fragments/9847.false_negative

```diff
@@ -0,0 +1,4 @@
+Fix a false negative when `isinstance` has too many arguments.
+Change now provides a `too-many-arguments` output with behavior similar to other `too-many-arguments` calls.
```

Comment:
Isn't it `too-many-function-args`, not `too-many-arguments`?

Reply:
Yes, changed it to `too-many-function-args` in both locations in that line.

My early commits had erroneously assigned this warning to `too-many-arguments` before I realized there was a `too-many-function-args` message option.